summary.tex

```diff
@@ -0,0 +1,16 @@
+\section{Summary}
+
+Rubin Observatory is putting in place a dedicated {\it Early Science Program} to ensure high-impact science during the first year of operation of the \lsst.
+This Early Science Program is motivated by the fact that in the current baseline there will be no science-ready data products released before \drone. 
+
+One of the four science pillars of LSST is time-domain astronomy, which is enabled by alerts on LSST detections of transient, variable, and/or moving objects. 
+Alerts are the only data product that will be immediately available (within 60 seconds of image readout) and publicly shareable, i.e not subject to a proprietary period \citep{LSE-163},  \citep{RDO-013}. 
+The worldwide community is actively preparing to process the LSST alert stream and use it to generate groundbreaking scientific results. Additionally, for many science goals, time-sensitive follow-up observations after discovery are crucial to take full advantage of the Rubin data. 
+
+A key component of the \esp is the capability to build {\it incremental} templates from on-sky imaging as it becomes available during commissioning and the early phases of the survey. 
+Such templates will be built periodically as images accumulate to allow for partial alert generation over an incomplete sky footprint.
+Where possible, templates will be built from all commissioning data before the start of year one and used to generate alerts during year one.
+How extensive these templates are at the start of full survey operations will be influenced on the overall success of commissioning.
```

Comment:
suggest 
How extensive these templates are at the start of full survey operations will influence the overall success of commissioning.

Reply:
I don't understand this. Generating templates is not a goal of commissioning so how does the state of the template library at the start of operations have any bearing on the success of commissioning?

Reply:
I did have a typo in my original sentence that might have been confusing.  on -> by